src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedTilesDAO.scala

```diff
@@ -0,0 +1,67 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import java.util.Date
+
+import com.datastax.spark.connector._
+import com.microsoft.partnercatalyst.fortis.spark.dto.ComputedTile
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SparkSession
+
+trait ComputedTilesDAOConfig extends Serializable {
+  def retrospectivePeriodCountPerType(): Map[String, Int]
+}
+
+case class DefaultComputedTilesDAOConfig() extends ComputedTilesDAOConfig {
+  override def retrospectivePeriodCountPerType(): Map[String, Int] = Map(
+    (PeriodType.Minute.periodTypeName, 4*60), /* 4 hours */
```

Comment:
Good question. The objective of this is to answer questions like "how far back should I go if I'm making a by-the-minute analysis" and in this case, this map would give an answer of "240 minutes" (4 hours).